hr_attendance_calendar/models/hr_attendance_day.py

```diff
@@ -0,0 +1,297 @@
+
+    # Working schedule
+    cal_att_ids = fields.Many2many(comodel_name='resource.calendar.attendance',
+                                   string="Working schedule",
+                                   required=True)
+    working_day = fields.Char(string="Working day",
+                              compute='_compute_working_day',
+                              readonly=True,
+                              store=True)
+
+    # Leaves
+    leave_ids = fields.Many2many(comodel_name='hr.holidays',
+                                 string='Leaves')
+    in_leave = fields.Boolean(string='In leave',
+                              compute='_compute_in_leave',
+                              store=True)
+
+    # Due hours
+    due_hours = fields.Float(string='Due hours',
+                             compute='_compute_due_hours',
+                             readonly=True,
+                             store=True)
+
+    # Attendances
+    attendance_ids = fields.One2many(string="Attendances",
+                                     comodel_name='hr.attendance',
+                                     inverse_name='attendance_day_id',
+                                     readonly=True)
+
+    # Worked
+    worked_hours = fields.Float(string='Worked hours',
+                                compute='_compute_worked_hours',
+                                store=True,
+                                readonly=True,
+                                oldname='calendar_worked_hours')
+    logged_hours = fields.Float(string='Logged hours',
+                                compute='_compute_logged_hours',
+                                store=True,
+                                readonly=True)
+
+    # Break
+    # due_break_min = fields.Float(string='Minimum break due',
+    #                              compute='_compute_due_break',
+    #                              store=True, )
+    # due_break_total = fields.Float(string='Total break due',
+    #                                compute='_compute_due_break',
+    #                                store=True, )
+    break_ids = fields.One2many(string='Breaks',
+                                comodel_name='hr.attendance.break',
+                                inverse_name='attendance_day_id',
+                                readonly=True,
+                                store=True)
+    break_max = fields.Float(string='Longest break',
+                             compute='_compute_break_max',
+                             store=True, )
+    break_total = fields.Float(string='Total break',
+                               compute='_compute_break_total',
+                               store=True, )
+
+    # Extra hours
+    extra_hours = fields.Float(string="Extra hours",
+                               compute='_compute_extra_hours',
+                               store=True,
+                               oldname='total_bonus_malus')
+    weighting = fields.Float(String="Weighting",
```

Comment:
Please add a help parameter to quickly describe what is this field because it's not very clear with the name.